examples/text_classification/predict.py

```diff
@@ -5,9 +5,21 @@
 from torchtext.datasets.text_classification import text_normalize
 
 
-def predict(text, model, dictionary):
+def predict(text, model, vocab):
```

Comment:
I think this example might not work anymore, since we moved "text_normalize" into the get_tokenizer abstraction